File: pfaProject/ASP.NET/PfaBackFinal/Models/Candidat.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace HdBackApp.Models
{
    public class Candidat
    {
        [Key]
        public int Id { get; set; }
        public string Nom { get; set; }
        public string Prenom { get; set; }
        public  DateTime DateDeNaissanse { get; set; }
        public string Email { get; set; }
        public string Ville { get; set; }
        public int Telephone { get; set; }
        public ICollection<Candidature> candidatures { get; set; }
        public Candidat()
        {
            candidatures = new Collection<Candidature>();
        }
    }
}

File: pfaProject/Angular/src/app/models/Candidat.ts
import { Candidature } from './Candidature';
export class Candidat {
    id:number;
    nom:string;
    prenom:string;
    dateDeNaissance:Date;
    email:string;
    ville:string;
    telephone:number;
    candidatures:Candidature[]=[];
   
}
